libgstd/libgstd.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * GStreamer Daemon - Gst Launch under steroids
+ * Copyright (c) 2015-2021 Ridgerun, LLC (http://www.ridgerun.com)
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+#ifndef __LIBGSTD_H__
+#define __LIBGSTD_H__
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <gst/gst.h>
+#include <glib-unix.h>
```

Comment:
I don't think we need this here, but in the daemon.

Reply:
This is necessary because the function `gstd_context_add_group` use in its parameters a `GOptionContext`. I understand it's not something for this PR, but in future we don't want users to include this headers as a min requirement to work with libgstd. What I did is to remove them from `gstd.c`